keyboards/thevankeyboards/minivan/keymaps/like_jis/keymap.c

```diff
@@ -0,0 +1,176 @@
+#include QMK_KEYBOARD_H
+#include "keymap_jp.h"
+
+extern keymap_config_t keymap_config;
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+extern uint8_t is_master;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_number {
+  _BASE = 0,
+  _LOWER,
+  _RAISE,
+  _ADJUST,
+};
+
+enum custom_keycodes {
+  LOWER = SAFE_RANGE,
+  RAISE,
+  ADJUST,
+  KANJI,
+  RGBRST
+};
+
+enum tapdances{
+  TD_SCCL = 0,
+};
+
+// Layer Mode aliases
+#define _____ KC_TRNS
+#define XXXXX KC_NO
+
+#define KC_TBSF  LSFT_T(KC_TAB)
+#define KC_ROSF  RSFT_T(KC_RO)
+#define KC_ALAP  LALT_T(KC_APP)
+
+// Layer tap
+#define KC_BSLO  LT(_LOWER, KC_BSPC)
+#define KC_SPRA  LT(_RAISE, KC_SPC)
+#define KC_MLAD  MO(_ADJUST)
+
+#define KC_SCCL  TD(TD_SCCL)
+
+qk_tap_dance_action_t tap_dance_actions[] = {
+  [TD_SCCL] = ACTION_TAP_DANCE_DOUBLE(KC_SCLN, KC_QUOT),
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_BASE] = LAYOUT( \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater and less "hack-y".